assets/vidstack_helper.js

```
@@ -1,187 +1,163 @@
+            translations = await response.json();
+        } catch (error) {
+            console.error('Fehler beim Laden der Übersetzungen:', error);
+            // Fallback zu leeren Übersetzungen, wenn das Laden fehlschlägt
+            translations = { de: {}, en: {} };
         }
-    };
+    }
 
     function getText(key, lang = 'en') {
-        return translations[lang][key] || key;
+        return (translations[lang] && translations[lang][key]) || key;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)